javascript/index.js

```diff
@@ -31,35 +34,66 @@ function printMilliseconds() {
 }
 
 function printSplit() {
-  // ... your code goes here
+  btnRight.addEventListener('click', () => {
+    if (btnRight.classList.contains('split')){
+    splits.innerHTML = splitValue
```

Comment:
Similar to the printMinutes and printSeconds comments, splitValue does not exist here, but you can use the chronometer splitClick method for this. you can call it simply doing: chronometer.splitClick()